res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1417,17 +1416,15 @@ PioneerDDJSX.tempoResetButton = function(channel, control, value, status, group)
 };
 
 PioneerDDJSX.autoLoopButton = function(channel, control, value, status, group) {
-    if (value) {
-        if (engine.getValue(group, "loop_enabled")) {
-            engine.setValue(group, "reloop_toggle", true);
-        } else {
-            engine.setValue(group, "beatloop_activate", true);
-        }
+    if (engine.getValue(group, "loop_enabled")) {
+        engine.setValue(group, "reloop_toggle", value);
+    } else {
+        engine.setValue(group, "beatloop_activate", value);
```

Comment:
The issue with storing loops is a [known bug](https://bugs.launchpad.net/mixxx/+bug/1596236).